RTNeural/gru/gru.h

```diff
@@ -96,6 +96,8 @@ class GRULayer final : public Layer<T>
     T* zVec;
     T* rVec;
     T* cVec;
+
+    static const int kNumBiasLayers { 2 };
```

Comment:
I think this could still be `constexpr`, no?

Reply:
You can’t compile `constexpr` variables inside of a class, IIRC. I tried compiling with that keyword and it didn’t work, so I have to use `static const` here.

Reply:
Oh weird... maybe try declaring it [like this](https://github.com/jatinchowdhury18/RTNeural/blob/main/RTNeural/gru/gru.h#L107)?

Reply:
That worked, thanks!